### PR TITLE
Add preview section for botbuilding process

### LINKED ISSUE #585: Change preview to stay on the screen rather than be added as a bot
**Actual Behaviour**

The preview of chatbot is added to screen like a bot.

**Expected Behaviour**

The preview of chatbot should stay on the screen.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by DNS-404**
Fixes #585 

Changes: Added preview section

Surge Deployment Link: https://pr-609-fossasia-susi-skill-cms.surge.sh

Screenshot:
screen shot 2018-06-08 at 11 16 18 am

### CODE REVIEW

File: src/components/BotBuilder/BotBuilder.js [hunk 1]
```diff
@@ -92,68 +92,74 @@ class BotBuilder extends React.Component {
...
+                                                <Deploy />
+                                            </Tab>
+                                        </Tabs>
+                                    </Col>
+                                    <Col xs={12} md={4} style={{textAlign:window.innerWidth>769?'right':'left', borderLeft:'1px solid rgb(66, 133, 244)', padding:'0.01em 16px'}}>
+                                        <br className='display-mobile-only'/>
+                                    <h2 style={{padding:'10px 0 10px 10px', textAlign:'left'}}>Preview</h2>
+                                    <iframe name="frame-preview" src="/BotPreview.html" width="350px" height="600px" frameBorder="1"></iframe>
+                                    {
+//                                    {!this.state.chatbotOpen?(<RaisedButton
```
Comment (hkedia321) [inline src/components/BotBuilder/BotBuilder.js]: remove these commented lines?

Comment (DNS-404) [reply src/components/BotBuilder/BotBuilder.js]: Just like I said.. "WIP" 😅 
I'll remove them as soon as I'm done.

**Comment by DNS-404** [comment]
Everyone, please review once I remove "WIP". I'm going to do all of these changes.

**Comment by DNS-404** [comment]
@hkedia321 @Akshat-Jain @PrP-11 @anshumanv @akshatnitd 
Please review. Please check functionality and code. Regarding responsiveness, the whole page isn't responsive and that needs to be done for all elements of the page.

**Comment by DNS-404** [comment]
@Akshat-Jain Done.

**Comment by DNS-404** [comment]
https://github.com/fossasia/susi_skill_cms/pull/628 - new PR for this

**Pull request closed without merging**